### DISCUSSION TITLE
Incorrect "numbered lists"

### DISCUSSION THREAD

**Discussion by kuraga** [community/community#109851] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

The next paragraph contains of the following source:

```
5. This is item "5".
7. This is item "7".
10. This is item "10".
```

The paragraph:

5. This is item "5".
7. This is item "7".
10. This is item "10".

Actual screenshot:

![image](https://github.com/community/community/assets/1063219/4314eac6-eee6-48a2-92c5-d4564bb20d41)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi there @kuraga, 

Would you mind sharing a bit more about where in GitHub you're experiencing this and if you're experiencing it across the platform? Thanks!

**Reply by kuraga** [reply]
Hi there @queenofcorgis ,

on this page? In the post above, I'm experiencing.  Aren't you?

**Reply by kuraga** [reply]
If "where else", I was experiencing at the Issues, Pull Requests.

**Reply by queenofcorgis** [reply]
@kuraga I am experiencing the same - I just tested it for myself. Thanks for letting us know - I've passed this along internally. Cheers!

**Reply by queenofcorgis** [reply, marked as answer]
@kuraga I chatted with some of our Markdown experts and this is actually expected and standard behavior - thanks for allowing me to learn something new. 

The reasoning is that it is more ergonomic - think in instances where you'd want to add an item between "first" and "second" then you don't have to change all the numbers - you just add another number and GitHub takes care of making it sequential. 

Here is the [markdown doc](https://www.markdownguide.org/basic-syntax/#ordered-lists) that share a bit more on the formatting.

**Reply by kuraga** [reply]
Ok, it follows the Standard.

**Comment by Vasa9999** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Feature Area
> 
> Issues
> 
> ### Body
> 
> The next paragraph contains of the following source:
> 
> ```
> 5. This is item "5".
> 7. This is item "7".
> 10. This is item "10".
> ```
> 
> The paragraph:
> 
> 5. This is item "5".
> 7. This is item "7".
> 10. This is item "10".
> 
> Actual screenshot:
> 
> ![image](https://github.com/community/community/assets/1063219/4314eac6-eee6-48a2-92c5-d4564bb20d41)
> 
> 
> ### Guidelines
> 
> - [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by KatyFrancesA** [comment]
Since you can't rely on the automatic Markdown numbering in Issues for this specific case, you have to bypass it and create the numbering manually.

<h3>Workarounds for You (to get the visual appearance you want)</h3>

**Manual Numbering with HTML Line Breaks (Most Reliable Visually):**

Treat each item as a separate line of text and force line breaks. This prevents Markdown from trying (and failing) to make a proper list.

`
5. This is item "5". <br>
7. This is item "7". <br>
10. This is item "10".
`

**_Result:_** This will display the text exactly as written, with the numbers 5, 7, and 10 visible, each on its own line. It won't have list indentation.

**Manual Numbering (Simpler, might have spacing issues):**

Type the lines directly. Sometimes Markdown adds extra paragraph spacing between lines like this, sometimes not.

`
5. This is item "5".
7. This is item "7".
10. This is item "10".
`

**_Result:_** Visually similar to option 1, but spacing might be less predictable than using **< br>**

**HTML List with value Attribute (Might work, might be stripped):**

You could try using the HTML **< li>** tag's value attribute. GitHub might respect this, but it also might strip it out for security/consistency.

```
<ol>
  <li value="5">This is item "5".</li>
  <li value="7">This is item "7".</li>
  <li value="10">This is item "10".</li>
</ol>
```

**_Result:_** If GitHub renders the value attribute, this would be the semantically correct way to get 5., 7., 10. with list indentation. If it ignores value, it will likely render as 1., 2., 3. or possibly 5., 6., 7.. Worth a quick test in the preview.

The bug is that GitHub Issues doesn't render 5. Item as 5. Item. The workaround is to manually type the 5. yourself and format the lines so Markdown doesn't try to auto-number them (using **< br>** is often best for this).
